contracts/azorius/LinearERC20Voting.sol

```diff
@@ -151,14 +151,16 @@ contract LinearERC20Voting is BaseStrategy, BaseQuorumPercent, BaseVotingBasisPe
             uint256 yesVotes,
             uint256 abstainVotes,
             uint32 startBlock,
-            uint32 endBlock
+            uint32 endBlock,
+            uint256 votingSupply
         )
     {
         noVotes = proposalVotes[_proposalId].noVotes;
         yesVotes = proposalVotes[_proposalId].yesVotes;
         abstainVotes = proposalVotes[_proposalId].abstainVotes;
         startBlock = proposalVotes[_proposalId].votingStartBlock;
         endBlock = proposalVotes[_proposalId].votingEndBlock;
+        votingSupply = _getProposalVotingSupply(_proposalId);
```

Comment:
`LinearERC20Voting` doesn't actually need this extra storage, right? Adding this is purely for the purpose of supporting the new `LinearERC20WrappedVoting` contract, right? I'm hesitant to add extra storage to this contract for the purpose of making inheritance work for the other one

Reply:
yeah `LinearERC20Voting`, but it's needed to snapshot it when a proposal is created for `LinearERC20WrappedVoting`

All the instances of `LinearERC20Voting` we create on the frontend wouldn't have this storage, only strategies for wrapped tokens would use `LinearERC20WrappedVoting`

Reply:
oh hmm wait I did put this on the `LinearERC20Voting`, why did I do that ha

Reply:
now I see what you're saying @tbwebb22